pykoi/rlhf/config.py

```diff
@@ -68,7 +66,7 @@ class RLHFConfig:
     #     default=8,
     #     metadata={"help": "Batch size."})
     per_device_train_batch_size: Optional[int] = field(
-        default=2, metadata={"help": "Batch size per device for training."}
+        default=1, metadata={"help": "Batch size per device for training."}
```

Comment:
nit: could you please make a TODO here that for trl 0.7.4 there is a OOM issue and we need to come back to revisit.

Reply:
added TODO